soc/ti/mspm0/mspm0l/Kconfig.soc

```diff
@@ -63,10 +63,12 @@ config SOC_MSPM0L1346
 config SOC_MSPM0L2227
 	bool
 	select SOC_SERIES_MSPM0L
+	select CPU_HAS_ARM_MPU
```

Comment:
this has no place here

Reply:
So, where and how should this be selected? Do we need to spin of MSPM0L222x as a separate series?

Reply:
put it in `Kconfig`. As per https://docs.zephyrproject.org/latest/hardware/porting/soc_porting.html#create-your-soc-directory
> Kconfig.soc: the base SoC configuration which defines a Kconfig SoC symbol in the form of config SOC_<soc-name> and provides the SoC name to the Kconfig SOC setting. If the soc.yml describes a SoC family and series, then those must also be defined in this file. Kconfig settings outside of the SoC tree must not be selected. To select general Zephyr Kconfig settings the Kconfig file must be used.